### PR TITLE
Show admins the sites this partner appears on

### LINKED ISSUE #2231: show information about which sites a partner appears on to all users with view permission for that partner
## User story

As a partner admin, partnership admin, or neighbourhood admin, it would be useful to see which sites the partner I admin for appears on.

## A/C
- [ ] If an admin can see an edit partner page, then they can see a list of sites this partner is visible on (as just text)

### PR DESCRIPTION
**Pull request by ivan-kocienski-gfsc**
Closes #2231 

This is just a list of links to the sites (as the original behaviour) not links to the partner page of that site.

### CODE REVIEW

File: app/views/admin/partners/edit.html.erb [hunk 1]
```diff
@@ -2,33 +2,34 @@
 <h1 class="page-header">Edit Partner: <em><%= @partner.name %></em></h1>
 
 <% if current_user.root? -%>
```
Comment (r-ferrier) [inline app/views/admin/partners/edit.html.erb:4]: this section of info is still hidden to everyone except root users - think we should probably just show all of this info to anyone who can view a partner in the admin!

**Review by r-ferrier — CHANGES_REQUESTED** [review]
Sorry, I appreciate this wasn't clear from the issue but I just meant for all the metadata at the top of the partner to be visible to all admin users, and not just roots. If you could remove the root restriction for the rest of it this will be good to go.

File: app/views/admin/partners/edit.html.erb (hunk 1 above)
Comment (ivan-kocienski-gfsc) [reply app/views/admin/partners/edit.html.erb:4]: but it's not part of the issue A/C. perhaps i could extend/alter the issue to include all this stuff?

do you also want the ID readout?

**Review by r-ferrier — APPROVED** [review]
Looks good!

File: app/views/admin/partners/edit.html.erb (hunk 1 above)
Comment (r-ferrier) [reply app/views/admin/partners/edit.html.erb]: I think you wrote the AC! I'll add it in.

**Pull request merged**